kornia/filters/sepia.py

```diff
@@ -0,0 +1,90 @@
+import torch
+import torch.nn as nn
+
+
+def sepia(input: torch.Tensor, rescale: bool = True) -> torch.Tensor:
+    r"""Apply to a tensor the sepia filter.
+
+    Args:
+        input: the input tensor with shape of :math:`(B, C, H, W)`.
+        rescale: If True, the output tensor will be rescaled (max values be 1. or 255).
+
+    Returns:
+        - torch.Tensor: The sepia tensor of same size and numbers of channels
```

Comment:
```suggestion
        The sepia tensor of same size and numbers of channels
```
type is inferred automatically